releasenotes/notes/mps_print_bond_dimension-6dc6f6cf0365c2ae.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    If the environment variable `MPS_OUTPUT_DATA` is set to 1, then MPS
+    log data is printed to the metadata. This includes, after every relevant
+    instruction, the bond dimensions and the values discarded during approximation.
```

Comment:
I suggest to add a couple of sentences explaining what you mean by "discarded values". Personally I misunderstood it, and discovered the correct meaning only when I read the code.

Reply:
Added a sentence both in release notes and in svd.cpp. Let me know if it makes more sense now.